src/kirby/components/card/card.component.ts

```diff
@@ -8,10 +11,32 @@ import { Component, OnInit, Input } from '@angular/core';
 export class CardComponent implements OnInit {
   @Input() title: string;
   @Input() subtitle: string;
+  sizes = {
+    ['card-small']: 360,
+    ['card-medium']: 720,
+    ['card-large']: 1024
+  };
 
-  constructor() { }
+  constructor(
+    private elementRef: ElementRef,
+    private resizeObserverService: ResizeObserverService,
+    private renderer: Renderer2) {
+  }
 
   ngOnInit() {
+    this.resizeObserverService.observe(this.elementRef, (entry) => this.handleResize(entry));
+  }
+
+  private handleResize(entry: ResizeObserverEntry) {
+    Object.entries(this.sizes).forEach(([size, width]) => {
+      if (entry.contentRect.width >= width) {
+          // this.renderer.setAttribute(entry.target, size, '');
```

Comment:
My internal linter tells me, that this code will never be run. :)
The same goes for the comment below.

Reply:
True :) We need to talk about whether to use css classes or attributes... Or maybe it doesn't really matter - but let's decide together

Reply:
For me styling belongs in css classes and not attributes. Would that not also be be more "valid" HTML way?

Is there any pros with one over the other?